index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="CSS/style.css" />
+    <title>Natnael Front-End Developer</title>
+</head>
+<body>
+    <div class="first">
+        <header class="header bg-white">
+          <nav class="navbar flex">
+            <a href="#" class="navbar__link link fz-20 fw-700 lh-20">N-A</a>
+            <div class="navbar__menu">
+              <ul class="navbar__menu__ul ul flex">
+                <li class="navbar__menu__li">
+                  <a href="#Portfolio" class="navbar__menu__link link fw-500 fz-15 lh-20 clr-n-600">Portfolio</a>
+                </li>
+                <li class="navbar__menu__li">
+                  <a href="#about" class="navbar__menu__link link">About</a>
+                </li>
+                <li class="navbar__menu__li">
+                  <a href="#contact" class="navbar__menu__link link">Contact</a>
+                </li>
+              </ul>
+            </div>
+            <img class="navbar__list" src="images/list.png" alt="list icon" />
+          </nav>
+        </header> 
+        <main class="head flex">
```

Comment:
- [ ] [optional] You can use ```section``` tag instead of ```main``` to wrap this section, this will make your code more semantic